Tasks/AzureRmWebAppDeploymentV4/operations/KuduServiceUtility.ts

```diff
@@ -313,14 +312,14 @@ export class KuduServiceUtility {
         }
     }
 
-    private async runCommand(physicalPath: string, command: string, timeOutInMinutes: number, pollFile: string): Promise<void> {
+    private async runCommand(physicalPath: string, command: string, timeOutInMinutes: number, pollFolderPath: string, pollFile: string): Promise<void> {
```

Comment:
Can we make pollFile and pollFolderPath optional parameters? This will avoid null parsing when polling is not required and maintain more clarity in function call.

Reply:
done. thanks